server/PlayLingua.Adapter.InMemoryDb/Repositories/BookRepository.cs

```diff
@@ -16,9 +17,38 @@ public BookRepository()
             _data.Add(new Book(Guid.NewGuid(), "Sample Book 2"));
         }
 
-        public List<Book> GetBooks()
+        public List<Book> List()
         {
             return _data;
         }
+
+        public Book Add(Book book)
+        {
+            var addedBook = new Book(Guid.NewGuid(), book.Name);
```

Comment:
You don't have to create a new book item here, you can directly use "book" request parameter

Reply:
But the id of book is not newGuid, it is something like 0000-00000-0000-00000